airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/b03a9f3e-22a5-11eb-adc1-0242ac120002.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sourceDefinitionId": "b03a9f3e-22a5-11eb-adc1-0242ac120002",
+  "name": "Mailchimp",
+  "dockerRepository": "airbyte/source-mailchimp",
+  "dockerImageTag": "latest",
```

Comment:
when we are ready to merge this we want to set this to the version that you specified in the dockerfile in this case 0.1.0. you can leave it as you have it for now while you are iterating on it still.